### PR TITLE
CDAP-2925: Document how to create dataset with properties better

### PR DESCRIPTION
**Pull request by jwang47**
https://issues.cask.co/browse/CDAP-2925
http://builds.cask.co/browse/CDAP-RBT381

### CODE REVIEW

File: cdap-docs/reference-manual/source/cli-api.rst [hunk 1]
```diff
@@ -200,11 +202,12 @@ These are the available commands:
    ``get worker runs <app-id.worker-id> [<status>] [<start-time>] [<end-time>] [<limit>]``,"Gets the run history of a Worker."
    ``get worker runtimeargs <app-id.worker-id>``,"Gets the runtime arguments of a Worker."
    ``get worker status <app-id.worker-id>``,"Gets the status of a Worker."
-   ``get workflow current <app-id.workflow-id> <runid>``,"Gets the currently running nodes of a Workflow for a given run id."
+   ``get workflow current <app-id.workflow-id> <runid>``,"Gets the currently running nodes of a workflow for a given run id."
```
Comment (awholegunch) [inline cdap-docs/reference-manual/source/cli-api.rst]: isn't "Workflow" consistent with the others?

File: cdap-docs/reference-manual/source/cli-api.rst [hunk 2]
```diff
@@ -200,11 +202,12 @@ These are the available commands:
...
    ``get workflow runs <app-id.workflow-id> [<status>] [<start-time>] [<end-time>] [<limit>]``,"Gets the run history of a Workflow."
    ``get workflow runtimeargs <app-id.workflow-id>``,"Gets the runtime arguments of a Workflow."
    ``get workflow schedules <app-id.workflow-id>``,"Resumes a schedule"
    ``get workflow status <app-id.workflow-id>``,"Gets the status of a Workflow."
+   ``get workflow token <app-id.workflow-id> <runid> [at node <workflow-node>] [scope <workflow-token-scope>] [key <workflow-token-key>]``,"Gets the workflow token of a workflow for a given run id."
```
Comment (awholegunch) [inline cdap-docs/reference-manual/source/cli-api.rst]: ibid.

File: cdap-docs/reference-manual/source/cli-api.rst [hunk 3]
```diff
@@ -215,35 +218,33 @@ These are the available commands:
...
-   ``set stream format <stream-id> <format> [<schema>] [<settings>]``,"Sets the format of a Stream. <schema> is a sql-like schema ""column_name data_type, ..."" or avro-like json schema and <settings> is specified in the format ""key1=v1 key2=v2""."
-   ``set stream notification-threshold <stream-id> <notification-threshold-mb>``,"Sets the Notification Threshold of a Stream."
-   ``set stream properties <stream-id> <local-file-path>``,"Sets the properties of a Stream, such as TTL, format, and notification threshold."
-   ``set stream ttl <stream-id> <ttl-in-seconds>``,"Sets the Time-to-Live (TTL) of a Stream."
+   ``set stream format <stream-id> <format> [<schema>] [<settings>]``,"Sets the format of a Stream. <schema> is a sql-like schema ""column_name data_type, ..."" or Avro-like JSON schema and <settings> is specified in the format ""key1=v1 key2=v2""."
+   ``set stream notification-threshold <stream-id> <notification-threshold-mb>``,"Sets the notification threshold of a Stream."
+   ``set stream properties <stream-id> <local-file-path>``,"Sets the properties of a stream, such as TTL, format, and notification threshold."
```
Comment (awholegunch) [inline cdap-docs/reference-manual/source/cli-api.rst]: Stream

File: cdap-docs/reference-manual/source/cli-api.rst [hunk 4]
```diff
@@ -263,7 +270,7 @@ These are the available commands:
    **Adapter Lifecycle**
    ``create adapter <adapter-name> <adapter-spec-file>``,"Creates an Adapter."
    ``delete adapter <adapter-name>``,"Deletes an Adapter."
-   ``describe adapter <adapter-name>``,"Lists all Adapters."
+   ``describe adapter <adapter-name>``,"Shows information about an adapter."
```
Comment (awholegunch) [inline cdap-docs/reference-manual/source/cli-api.rst]: Adapter

File: cdap-docs/reference-manual/source/cli-api.rst [hunk 5]
```diff
@@ -149,12 +151,12 @@ These are the available commands:
    ``delete preferences worker [<app-id.worker-id>]``,"Deletes the preferences of a Worker."
    ``delete preferences workflow [<app-id.workflow-id>]``,"Deletes the preferences of a Workflow."
    ``delete stream <stream-id>``,"Deletes a Stream."
-   ``deploy app <app-jar-file>``,"Deploys an Application."
+   ``deploy app <app-jar-file> [<app-config>]``,"Deploys an Application optionally with a serialized configuration string."
```
Comment (awholegunch) [inline cdap-docs/reference-manual/source/cli-api.rst]: Deploys an Application, optionally with a serialized configuration string.

**Comment by awholegunch** [comment]
A few comments, otherwise LGTM.

File: cdap-docs/reference-manual/source/cli-api.rst (hunk 1 above)
Comment (jwang47) [inline cdap-docs/reference-manual/source/cli-api.rst]: This is generated.

Comment (jwang47) [inline cdap-docs/reference-manual/source/cli-api.rst]: Also, it's supposed to be "workflow" now.

**Comment by jwang47** [comment]
All the cli-api.rst parts you commented are were generated. We can modify those in another PR.

**Comment by awholegunch** [comment]
OK. Please open another PR, copy the changes, and then fix this one...right?

**Comment by jwang47** [comment]
I just removed the cli-api.rst changes. We can fix cli-api.rst in a separate PR since it's not related to this change.

**Pull request merged**